### PR TITLE
EVG-20813: Update patch_with_aborted_task 

### PR DESCRIPTION
**Pull request by SupaJoon**
EVG-20813

### Description
Update the task and version utilized in this test to avoid an edge case described in this comment:
https://github.com/evergreen-ci/spruce/pull/2006#discussion_r1310541344
Depends on https://github.com/evergreen-ci/evergreen/pull/7031

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "spruce",
-  "version": "3.0.134",
+  "version": "3.0.135",
```
Comment (khelif96) [inline package.json:3]: You need to merge main into the base branch so that this isn't included as part of the diff.

**Review by khelif96 — APPROVED** [review]
